src/modules/articles.js

```diff
@@ -1,7 +1,15 @@
 import axios from "axios";
 
-const fetchAllArticles = async () => {
-  let response = await axios.get("/articles");
-  return response.data.articles;
+const Articles = {
+  async index() {
+    let result = await axios.get("/articles");
+    return result.data.articles;
+  },
+
+  async show(articleId) {
+    let result = await axios.get(`/articles/${articleId}`);
+    return result.data.article;
```

Comment:
You need to take care of a potential error if you dont get a 200, basically the error message you’ve implemented in the backend